pkg/scheduler/framework/plugins/selectorspread/BUILD

```diff
@@ -12,6 +12,8 @@ go_library(
         "//staging/src/k8s.io/api/core/v1:go_default_library",
         "//staging/src/k8s.io/apimachinery/pkg/labels:go_default_library",
         "//staging/src/k8s.io/apimachinery/pkg/runtime:go_default_library",
+        "//staging/src/k8s.io/client-go/listers/core/v1:go_default_library",
+        "//staging/src/k8s.io/client-go/listers/apps/v1:go_default_library",
```

Comment:
I guess you manually updated this file? Bazel would be unhappy on this as "apps" should be alphabetically ahead of "core".

Reply:
Done.